config/src/config/network_config.rs

```diff
@@ -167,20 +167,28 @@ impl NetworkConfig {
         }
     }
 
-    pub fn load(&mut self, role: RoleType) -> Result<(), Error> {
+    /// Per convenience, so that NetworkId isn't needed to be specified for `validator_networks`
+    pub fn load_validator_network(&mut self) -> Result<(), Error> {
+        self.network_id = NetworkId::Validator;
+        self.load()
+    }
```

Comment:
doesn't ignoring the network_id like this kind of defeat the purpose of checking the invariants? i know this is the same behavior as before, but maybe we can make this safer

Reply:
So, I tried changing this, because I agree with you.

However, this is used as basically a "skip defining the validator network type" method.  So, without changing every validator config, it would crash if I added this here.  It's more about the fact that since we share a config type between the two that makes this rough.